Coordinators/Coordinators/Coordinators/AppCoordinator.swift

```diff
@@ -0,0 +1,80 @@
+// AppCoordinator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import UIKit
+
+/// Фукнции с помощью которых осуществляется переход между экранами
+final class AppCoordinator: NSObject, Coordinator, UINavigationControllerDelegate {
```

Comment:
```suggestion
final class AppCoordinator: NSObject, Coordinator {
```
ту мач, вынеси в extension